### PR TITLE
(version-bumping-update) Updates to version bumping.

### PR DESCRIPTION
**Pull request by rob-mongodb**
Staged: https://docs-mongodbcom-staging.corp.mongodb.com/meta/robertjustice/version-bumpiing-update/tutorials/version-bumping.html

### CODE REVIEW

File: source/tutorials/version-bumping.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ Step 1: Branch
 
 .. code-block:: sh
 
-   git checkout -b v2.2
+   git checkout -b v2.2 upstream/master
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/tutorials/version-bumping.txt]: Fixed.

File: source/tutorials/version-bumping.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@ Step 1: Branch
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/tutorials/version-bumping.txt]: Fixed.

File: source/tutorials/version-bumping.txt [hunk 3]
```diff
@@ -59,10 +59,13 @@ will find a file called  ``build_conf.yaml``. Update this file's
 Step 3: Update published branches configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In the ``docs-tools`` repository under the ``data/`` directory, you will
-find a file called ``bi-connector-published-branches.yaml``. Update this
-file to include a ``2.3`` version and a ``v2.2`` branch; and set the
-``version.upcoming`` key to ``2.3``.
+a. Checkout the master branch.
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/tutorials/version-bumping.txt]: Fixed.

File: source/tutorials/version-bumping.txt [hunk 4]
```diff
@@ -59,10 +59,13 @@ will find a file called  ``build_conf.yaml``. Update this file's
 Step 3: Update published branches configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In the ``docs-tools`` repository under the ``data/`` directory, you will
-find a file called ``bi-connector-published-branches.yaml``. Update this
-file to include a ``2.3`` version and a ``v2.2`` branch; and set the
-``version.upcoming`` key to ``2.3``.
+a. Checkout the master branch. 
+b. In the ``docs-tools`` repository under the ``data/`` directory, open
+   a file called ``bi-connector-published-branches.yaml``.
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/tutorials/version-bumping.txt]: Fixed.

**Comment by rob-mongodb** [comment]
@jdestefano-mongo , updated and ready for another look!

File: source/tutorials/version-bumping.txt [hunk 5]
```diff
@@ -36,11 +36,15 @@ been removed, and users should be redirected to ``reference/mongosqld``.
...
+From the project's repository, create a new branch for the current
+version:
+
 .. code-block:: sh
 
-   git checkout -b v2.2
+   git checkout -b v2.2 upstream/master
    git push origin v2.2
    git checkout master
+   git pull --rebase
```
Comment (jdestefano-mongo) [inline source/tutorials/version-bumping.txt]: Sorry if I was unclear: I meant on line 45:

```
git checkout -b v2.2 upstream/master
git pull --rebase
git push origin v2.2
git checkout master
```

Comment (rob-mongodb) [reply source/tutorials/version-bumping.txt]: Fixed.

**Comment by rob-mongodb** [comment]
@jdestefano-mongo , updated and ready for another look.

**Comment by jdestefano-mongo** [comment]
Merged and published. Thanks Rob!

**Pull request merged**